style.css

```diff
@@ -0,0 +1,123 @@
+@import url(https://fonts.googleapis.com/css?family=Playfair+Display|Poppins);
+
+body {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+.a-logo {
+  display: flex;
+  justify-content: end;
+  height: 100%;
+  width: 100%;
+}
+
+nav a {
+  display: none;
+}
+
+header > * {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo-container {
+  height: 100%;
+  width: 100%;
+}
+
+.fa-bars {
+  width: 300px;
+  color: black;
+  height: 100%;
+  margin-left: 90%;
+  margin-top: -50px;
+  cursor: pointer;
+  outline: none;
+  font-size: 30px;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+  flex-wrap: wrap;
+}
```

Comment:
- [ ] DRY stands for `Don't Repeat Yourself`, however you are repeating ourselves in your code, `display:flex , align-items`  `justify-content: space-evenly` `align-items: center` `flex-wrap: wrap`.

How do you keep CSS dry?

   DRY CSS comes down to 3 things:
- Group reusable css properties together.
- Name these groups logically.
- Add your selectors to the various css groups.

You can make a class and call it `.d-flex` to be called wherever you need that action